WORKSPACE

```diff
@@ -83,7 +83,7 @@ http_archive(
 )
 
 # For manual testing against an LLVM toolchain.
-# Use --crosstool_top=@llvm_toolchain//:toolchain
+# Use --extra_toolchains=@llvm_toolchain//:toolchain
```

Comment:
Is this still the right target (`toolchain`, not `cc_toolchain`)?

Reply:
Good call. It was previously pointing to a cc_toolchain_suite. Fixed it to point to `toolchain` targets.